source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -166,8 +166,8 @@
   "settings.support.reportProblem.downloadLogsProgressMessage": "ダウンロード用のログを準備しています",
   "settings.support.reportProblem.downloadLogsSuccessMessage": "ログのダウンロードに成功しました。",
   "settings.support.reportProblem.linkUrl": "https://iohk.zendesk.com/hc/ja/requests/new/",
-  "settings.support.steps.downloadLogs.description": "サポートチームが問題を調査しやすくするために、ここからログをダウンロードして、ダウンロードしたファイルをサポートリクエストの送信時に添付してください。ログに{stepsDownloadLogsLink}",
-  "settings.support.steps.downloadLogs.link": "機密情報は含まれていません。",
+  "settings.support.steps.downloadLogs.description": "サポートチームが問題を調査しやすくするために、{stepsDownloadLogsLink}して、ダウンロードしたファイルをサポートリクエストの送信時に添付してください。ログに機密情報は含まれていません。",
```

Comment:
@DeeJayElly are you sure we don't need a space after `{stepsDownloadLogsLink}` part?

Reply:
Hmm i was just looking at the screenshot from the task and don't think we have a space before or after the {stepsDownloadLogsLink}?

Reply:
Alright then :)